README.md

```diff
@@ -0,0 +1,54 @@
+# Testplane testing library
```

Comment:
Let's adopt a unified README style that we use across our repos. Here are logos for this repo:
- dark: https://nda.ya.ru/t/QpghjQix7H5MFD
- light: https://nda.ya.ru/t/ZDKyQ37U7H5MNo

Other things like badges you may copy from our mcp repo.

Reply:
Fixed for logo